src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -16,14 +17,20 @@
      */
     public static Command parse(String fullCommand) {
         String taskType;
-        String args;
+        String args = null;
         String[] words;
         words = fullCommand.split(" ",2);
```

Comment:
Because of the second parameter being 2, would the parsing of fullCommand be limited to only two strings?

Reply:
Yeah, it is true that fullCommand will be divided into two strings  at the beginning (The first String is `taskType` and the second String is `args`) And the second string will be processed later(maybe will continue to split to more strings):
The variable String `args` in `parse` method stands for all the arguments(parameter) of a fullCommand. e.g.:
in `add id/CS2113 s/19-20 Semester 2`, `args` is `id/CS2113 s/19-20 Semester 2` and there are two args(`id/CS2113` and `s/19-20 Semester`) in this `args` String. The args String will be processed (split into two String) in `processAddCommand(String args)`. I think maybe it is better to split parameters in `proccessXCommand` method because different commands needs to split the parameters in different way.

Reply:
Ohhhh okok I see then that's fine. Thanks!!